### PR TITLE
Fix NRE when combobox was disposed and Text set

### PR DESCRIPTION
**Pull request by sancheolz**
If combobox already was disposed and we setting Text field there will be NullReferenceException. On .net winforms there no excetions.

### CODE REVIEW

File: mcs/class/System.Windows.Forms/Test/System.Windows.Forms/ComboBoxTest.cs [hunk 1]
```diff
@@ -1500,7 +1500,39 @@ public void SortTest ()
 			cmb.Items.Add (new ComboVal ("A"));
 			cmb.Sorted = true;
 		}
-	}
+
+        [Test]
+        public void SetTextAfterDisposeTest()
+        {
+            try
```
Comment (marek-safar) [inline mcs/class/System.Windows.Forms/Test/System.Windows.Forms/ComboBoxTest.cs]: The try/catch is redundant

Comment (sancheolz) [reply mcs/class/System.Windows.Forms/Test/System.Windows.Forms/ComboBoxTest.cs]: Fixed

**Comment by sancheolz** [comment]
@marek-safar could you backport it to 2019-02

**Comment by marek-safar** [comment]
@monojenkins backport 2019-02

**Pull request merged**